src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -132,7 +177,7 @@ public void basicTest() throws Exception {
             .put(IDP_METADATA_URL, mockSamlIdpServer.getMetadataUri())
             .put("kibana_url", "http://wherever")
             .put("idp.entity_id", mockSamlIdpServer.getIdpEntityId())
-            .put("exchange_key", "abc")
+            .put("exchange_key", signingKeyB64Encoded)
```

Comment:
Why did we need to change this key? Seems like it should be the same as it was before the change?

Reply:
Fixed. I just reverted back. I was changing this cuz I was removing the padding function, so the key was forced to be at least 256 bits long. However, we brought back the padding function, so that this should be fine now.